src/components/table/LightTable.jsx

```diff
@@ -45,216 +41,153 @@ const styles = (theme) => ({
-    this.setState({ page }); // offset
-    handleEnumeration(rowsPerPage, offset, orderBy, orderDirection, searchString);
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
   };
 
-  handleSearchString = (event) => {
-    const { handleEnumeration } = this.props;
-    const { page, orderDirection, orderBy, rowsPerPage } = this.state; // limit
+  useEffect(() => {
     const offset = page * rowsPerPage;
-    const searchString = event.target.value;
-    this.setState({ searchString }); // filter
-    handleEnumeration(rowsPerPage, offset, orderBy, orderDirection, searchString);
-  };
-
-  render() {
-    const {
-      classes,
-      columns,
-      itemType,
-      rows,
-      summary,
-      totalCount,
-      filteredCount,
-      rowKey,
-    } = this.props;
-    const { orderBy, orderDirection, page, rowsPerPage } = this.state;
-    const ROW_HEIGHT = 50;
-    const ROWS_PER_PAGE = [5, 10, 25];
-    const emptyRows =
+    setEmptyRows(
       rowsPerPage < filteredCount
         ? rowsPerPage - Math.min(rowsPerPage, filteredCount - page * rowsPerPage)
-        : 0;
-    return (
-      <div>
-        {!summary && (
-          <div className={classes.search}>
-            <div className={classes.searchIcon}>
-              <SearchIcon />
-            </div>
-            <InputBase
-              placeholder="Search"
-              classes={{
-                root: classes.inputRoot,
-                input: classes.searchInput,
-              }}
-              onChange={this.handleSearchString}
-            />
-          </div>
-        )}
-        <Paper className={classes.root}>
-          <Table className={classes.table}>
-            <LightTableHead
-              columns={columns}
-              onRequestSort={this.handleRequestSort}
-              orderDirection={orderDirection}
-              orderBy={orderBy}
-              summary={summary}
-            />
-            <TableBody>
-              {rows && rows.length > 0 ? (
-                rows.map((row, index) => {
-                  const darkRow = index % 2 !== 0;
-                  return (
-                    <TableRow
-                      key={rowKey ? rowKey(row) : row.id}
-                      className={clsx({
-                        [classes.row]: true,
-                        [classes.darkRow]: darkRow,
-                        [classes.lightRow]: !darkRow,
-                      })}
-                    >
-                      {columns.map((col) => (
-                        <TableCell key={col.property} style={{ wordBreak: 'break-word' }}>
-                          {col.render ? col.render(row) : row[col.property]}
-                        </TableCell>
-                      ))}
-                    </TableRow>
-                  );
-                })
-              ) : (
-                <TableRow className={classes.row}>
-                  <TableCell colSpan={columns.length}>
-                    No {itemType} have been created yet
```

Comment:
Making a quick note so it's top of mind: if we filter down to no found datasets or snapshots, we show this, instead of `No {itemType} match your filter` (which I could have sworn we fixed when we added the totalRows field but perhaps not)

Reply:
ok, fix pushed.